src/cbc_sdk/platform/policies.py

```diff
@@ -0,0 +1,1222 @@
+    """
+    urlobject = "/policyservice/v1/orgs/{0}/policies"
+    urlobject_single = "/policyservice/v1/orgs/{0}/policies/{1}"
+    primary_key = "id"
+    swagger_meta_file = "platform/models/policy.yaml"
+    VALID_PRIORITIES = ["LOW", "MEDIUM", "HIGH", "MISSION_CRITICAL"]
+    VALID_SENSOR_SETTINGS = ["SHOW_UI", "ALLOW_UNINSTALL", "ALLOW_UPLOAD", "QUARANTINE_DEVICE", "ENABLE_FORENSICS",
+                             "LOGGING_LEVEL", "QUARANTINE_DEVICE_MESSAGE", "ENABLE_THREAT_SHARING", "SET_SENSOR_MODE",
+                             "SENSOR_RESET", "BLOCK_REMOVABLE_MEDIA", "POLICY_ACTION_OVERRIDE", "BACKGROUND_SCAN",
+                             "RATE_LIMIT", "QUEUE_SIZE", "DROP_CONNECTION_TIME", "CONNECTION_LIMIT", "LEARNING_MODE",
+                             "SET_AV_MODE", "SCAN_NETWORK_DRIVE", "BYPASS_AFTER_RESTART_MINS",
+                             "BYPASS_AFTER_LOGIN_MINS", "HELP_MESSAGE", "SHOW_FULL_UI", "SCAN_EXECUTE_ON_NETWORK_DRIVE",
+                             "DELAY_EXECUTE", "ALLOW_INLINE_BLOCKING", "PRESERVE_SYSTEM_MEMORY_SCAN", "HASH_MD5",
+                             "SCAN_LARGE_FILE_READ", "SECURITY_CENTER_OPT", "CB_LIVE_RESPONSE", "UNINSTALL_CODE",
+                             "ALLOW_EXPEDITED_SCAN", "UBS_OPT_IN", "DISABLE_MALWARE_SERVICES"]
+    VALID_SCAN_MODES = ["NORMAL", "AGGRESSIVE"]
+    VALID_SCAN_OPTIONS = ["AUTOSCAN", "DISABLED"]
+    VALID_DAYS = ["MONDAY", "TUESDAY", "WEDNESDAY", "THURSDAY", "FRIDAY", "SATURDAY", "SUNDAY"]
+
+    def __init__(self, cb, model_unique_id=None, initial_data=None, force_init=False, full_doc=False):
+        """
+        Initialize the Policy object.
+
+        Required Permissions:
+            org.policies (READ)
+
+        Args:
+            cb (BaseAPI): Reference to API object used to communicate with the server.
+            model_unique_id (int): ID of the policy.
+            initial_data (dict): Initial data used to populate the policy.
+            force_init (bool): If True, forces the object to be refreshed after constructing.  Default False.
+            full_doc (bool): If True, object is considered "fully" initialized. Default False.
+        """
+        super(Policy, self).__init__(cb, model_unique_id=model_unique_id, initial_data=initial_data,
+                                     force_init=force_init if initial_data else True, full_doc=full_doc)
+        self._object_rules = None
+        self._object_rules_need_load = True
+        self._info["version"] = 2
+        if model_unique_id is None:
+            self.touch(True)
+
+    class PolicyBuilder:
+        """
+        Builder object to simplify the creation of new Policy objects.
+
+        To use, call Policy.create() to get a PolicyBuilder, use its methods to set the properties of the
+        new policy, call its build() method to build the populated Policy, then call the policy save() method.
+        The org.policy(CREATE) permission is required.
+
```

Comment:
Somewhere in this section include something like
For information on values for policy settings including enumeration values, see the Policy Service API page:
https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/policy-service/#fields